### ISSUE TITLE
Focus lost in quantity input

### ISSUE THREAD

**Issue by marlass** [SAP/cloud-commerce-spartacus-storefront#994]
When interacting with quantity input field using the keyboard there is some weird behavior regarding focus management. Changing input value and waiting for few milliseconds will trigger product update and then focus will be lost on page. Also focus is not visible on +/- buttons. You are not able to see where you are, because of that.

**Comment by RadhepS** [comment]
PR: GH-2493
